Add Station M3 RK3588S Support

mo123 opened the pull request:
Station M3 RK3588S mini pc is based on the ROC-RK3588S-PC board.

dmanlfc commented:
I take it you haven't built this & tested it as you're missing some configuration.
also, why are you using the `orangepi 5` bootloader & uboot?

mo123 commented:
> I take it you haven't built this & tested it as you're missing some configuration. also, why are you using the `orangepi 5` bootloader & uboot?

Hi

I fixed the missing build option.

Tested and booting correctly.

I see several RK3588 devices use each others u-boot to reduce code duplication.
eg. mekotronics-r58 uses Rock5B u-boot.
ROC-RK3588-PC u-boot is compatible and can boot with the Orange-Pi 5 one.

The u-boot in Batocera Linux is still based on stvhay's code which is very old and I don't want to introduce extra 3rd party Github repos that might get forgotten about and not get updated.
For Armbian this u-boot is used - https://github.com/150balbes/u-boot-rk/tree/rk3588-next
Perhaps when u-boot gets updated for all RK3588 devices in Batocera Linux a different approach can be taken in future.

Replaced by PR 10980
